sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -88,6 +98,7 @@ case class CreateFunctionCommand(
         // For a permanent, we will store the metadata into underlying external catalog.
         // This function will be loaded into the FunctionRegistry when a query uses it.
         // We do not load it into FunctionRegistry right now.
+        checkIfResourceExists
```

Comment:
How about this?
```
        // Checks if the given resources exist
        val hadoopConf = sparkSession.sparkContext.hadoopConfiguration
        val nonExistentResources = resources.filter { r =>
          val path = new Path(r.uri)
          !path.getFileSystem(hadoopConf).exists(path)
        }
        if (nonExistentResources.nonEmpty) {
          throw new AnalysisException(s"Resources not found: ${nonExistentResources.mkString(",")}")
        }
```
btw, this check should be placed in the start of  `CreateFunctionCommand.run`?